notebooks/MAST/downloading_transiting_planet_data/downloading_exoplanet_data.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>Is there a chance more data will be added that will make your "two files" statement incorrect? You could add "at the time of the writing of this tutorial" or just try to avoid quantifying the number in the Markdown cell.</p><p> </p><p>Likewise, you want to "download one of them" and it's hardcoded in the next code block that it will be the first one. But I'm worried if the order of the filenames change in the future, that could break the rest of the tutorial. When I encountered this with a JWST related tutorial I just hardcoded the filename I wanted. I think it's fair to just explain to the user why that was done.</p><p> </p><p>It would be good to point out (here and elsewhere) how the URLs are formatted. It's interesting that it's in a subdirectory of the correctly formatted planet name. You can't JUST use the filename.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>